src/map/status.c

```diff
@@ -3482,31 +3482,31 @@ static void status_calc_regen_rate(struct block_list *bl, struct regen_data *reg
 	}
 
 
+	if (sc->data[SC_MAGNIFICAT]) {
+#ifndef RENEWAL // HP Regen applies only in Pre-renewal
+		regen->rate.hp += regen->rate.hp * 100 / 100;
+#endif
+		regen->rate.sp += regen->rate.sp * 100 / 100;
+	}
+
 	// Tension relax allows the user to recover HP while overweight
 	// at 1x speed. Other SC ignored? [csnv]
 	if (sc->data[SC_TENSIONRELAX]) {
 		if (sc->data[SC_WEIGHTOVER50] || sc->data[SC_WEIGHTOVER90]) {
 			regen->flag &= ~RGN_SP;
-			regen->rate.hp = 100;
+			regen->rate.hp += regen->rate.hp * 100 / 100;
 		} else {
-			regen->rate.hp += 200;
+			regen->rate.hp += regen->rate.hp * 200 / 100;
```

Comment:
Is this supposed to interact this way with Magnificat? The bonus seems very high here. If Magnificat is on, then these lines have been executed:

```
regen->rate.hp = 100; // line 3542
regen->rate.hp += regen->rate.hp * 100 / 100; // line 3487
```

and `regen->rate.hp` is currently `100 + 100 * 100 / 100 == 200`. If you do:

```
regen->rate.hp += regen->rate.hp * 200 / 100;
```

it means `200 + 200 * 200 / 100 == 600`. Are Tension Relax and Magnificat supposed to multiply each others this way? It feels to me that the effect of regen bonus is way too amplified after this change. (and the same applies to everything else below)